community/blog/2023-01 Role-based authentication in Azure API Management made easy in Arcus Scripting v0.8/README.md

```diff
@@ -0,0 +1,63 @@
+# Role-based authentication in Azure API Management made easy in Arcus Scripting v0.8
+Managing roles during JWT validation was tedious and repetitive work. Luckily, the newest Arcus Scripting release changed that.
+
+## Validating JSON web tokens in Azure API Management
+The `validate-jwt` policy in Azure API Management provides the capability to enforce a valid JWT in an incoming HTTP request. One of these validation rules is whether certain roles have been assigned to a service principal. This authorization functionality is very useful to allow/deny certain functionality to users of the application.
```

Comment:
I would change the 'certain roles have been assigned to a service principal part' to something like this:

```suggestion
The `validate-jwt` policy in Azure API Management provides the capability to enforce a valid JWT in an incoming HTTP request. One of these validation rules is whether the received token contains certain role claims. This authorization functionality is very useful to allow/deny certain functionality to users of the application based on their role membership.
```